### DISCUSSION TITLE
facing issue with oracle Materialized view refresh from airflow

### DISCUSSION THREAD

**Discussion by MaruvadaKameswaraRao** [apache/airflow#25459] [Q&A]

**Comment by potiuk** [comment]
Likely not if you won't explain more details. 

It's completely illegible what you try to do, please explain it better, show some examples and explain what does not work if you want someone to help. The way you asked it makes it impossible to even look at it and you are very much falling into https://dontasktoask.com/  - first spend time on explaining what you really want, before asking somone to spend their free time on trying to help you.
